.github/workflows/fii.yaml

```diff
@@ -4,8 +4,6 @@ on:
   push:
     tags:
       - '*'
-  pull_request:
```

Comment:
only build ffi on tags - fails on non-tagged PRs

Reply:
But does this mean that FFI building is no longer validated on PR builds?

Reply:
sorted. `goreleaser build` on PR & `goreleaser release` on tag